### PR TITLE
Update MET Filters

### PR DESCRIPTION
**Pull request by jkarancs**
Content of the PR:
- Update to latest Run II MET filter recommendations for 2016 data, found here:
  https://twiki.cern.ch/twiki/bin/view/CMS/MissingETOptionalFiltersRun2#How_to_run_the_Bad_Charged_Hadro
- Update recipe (need to merge a branch from the JetMET POG)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,11 +23,12 @@ setenv CMSSW_GIT_REFERENCE /cvmfs/cms.cern.ch/cmssw.git.daily
 git cms-init
 
 ```
- * Top branch needed to run updated double b tagger training on AK8 jets
+ * Top branch needed to run updated double b tagger training on AK8 jets, and latest Run II MET filters
 ```
 git fetch --tags btv-cmssw
 
 git cms-merge-topic -u cms-btv-pog:BoostedDoubleSVTaggerV3-WithWeightFiles-v1_from-CMSSW_8_0_8_patch1
```
Comment (alefisico) [inline README.md]: I was wondering if we still need this merge. BTW, which CMSSW release are you using? I was working in 8_0_20.

Comment (jkarancs) [reply README.md]: I don't think these producers are added to CMSSW yet. I didn't change the CMSSW version in the recipe, I kept 8_0_16. But I used the official JetMET pog recipe, so I think this is the correct way to do.

Comment (dmajumder) [reply README.md]: @jkarancs : thanks for the MET filter implementation. PR#55 is still open, do we merge it, or wait for these filters to be implemented in a later CMSSW release? It says that the bad muon and badCharged hadron filters are RECO problems, so these should disappear in the re-RECO, right?

Comment (jkarancs) [reply README.md]: @dmajumder 
In the mean time Savvas sent a PR that got merged, so I need to do this addition on top of that, so this PR should not be merged any more (conflict), I close it.
To answer your question more specifically, I think we should be able to use the latest MET recommendations whatever they might be, even if they are temporary. I think the filters are going to be implemented eventually in the Rereco, but until then, we should use this temporary recipe from the JetMET POG.

**Comment by jkarancs** [comment]
Dear all, I fixed the merge conflict so I can repoen this PR.

**Pull request merged**